tests/cloudferrylib/os/actions/test_check_networks.py

```diff
@@ -14,16 +14,35 @@
 #    under the License.
 
 
+import os
+
 import mock
 
 from cloudferrylib.base import exception
+from cloudferrylib.utils import utils
 from cloudferrylib.os.actions import check_networks
 from tests import test
 
 
+RESULT_FILE = 'tests/check_net_result'
```

Comment:
Updating log message either of the following line: 46,660,666 with info that network information is taken from Source or Destination, will make log more readable.